src/libworkbench/workbench-completion-provider.c

```diff
@@ -0,0 +1,188 @@
+
+#include <gio/gio.h>
+#include <gtksourceview/gtksource.h>
+
+#include "libworkbench-enums.h"
+#include "workbench-completion-provider.h"
+#include "workbench-completion-request.h"
+
+
+/**
+ * WorkbenchCompletionProvider:
+ *
+ * A base class for completion providers in Workbench.
+ *
+ * Since: 45.0
```

Comment:
GNOME 45 is still 3 months away :disappointed: 

but I guess it's not very important here if we release this under 44.x

Reply:
Oh, yeah that's just habit from writing for public libraries. `libworkbench`'s version is actually `0.{meson.project_version()}`, e.g. `0.44.2`.